mkt/extensions/views.py

```diff
@@ -85,6 +86,15 @@ class ExtensionViewSet(CORSMixin, MarketplaceView,
     queryset = Extension.objects.all()
     serializer_class = ExtensionSerializer
 
+    def filter_queryset(self, qs):
+        if self.action == 'list':
+            # The listing API only allows you to see extensions you've
+            # developed.
```

Comment:
The public API will be search, like for apps (which have the same restriction about listing)